tools/qdmanage

```diff
@@ -24,7 +24,9 @@ import json
 import re
 from collections.abc import Mapping, Sequence
 
-import qpid_dispatch_site  # noqa: F401 # imported for side-effects (inserts to sys.path)
+import qpid_dispatch_site
+qpid_dispatch_site.populate_pythonpath()
```

Comment:
Can we call the populate_pythonpath() in the init of qpid_dispatch_site so that we don't have to specifically call it ?

Reply:
Yes, but then we would have it called every time the file is included. Sometimes, you want to include the file and not add into pythonpath.

Also, importing for side-effects only is strange, I thought making this into explicit call is an improvement.

Reply:
I agree that if we need to include the file without affecting the pythonpath, yours is a good approach. Is there a place where this file is being imported where you don't need the pythonpath changed?

Reply:
pythonpath is only changed for qdstat and qdmanage; in all other uses (they are all new, in this PR) PYTHONPATH is not changed.